packages/vscode-extension/src/webview/components/UrlSelectItemGroup.tsx

```diff
@@ -0,0 +1,57 @@
+import React from "react";
+import UrlSelectItem from "./UrlSelectItem";
+import { UrlSelectFocusable } from "./UrlSelect";
+import { NavigationHistoryItem } from "../../common/Project";
+
+interface UrlSelectItemGroupProps {
+  items: NavigationHistoryItem[];
+  itemsRef: React.RefObject<HTMLDivElement>[];
+  refIndexOffset?: number;
+  textfieldRef: React.RefObject<HTMLInputElement>;
+  width: number;
+  onClose: (id: string) => void;
+  onNavigate: (
+    e: React.KeyboardEvent,
+    prev?: UrlSelectFocusable,
+    next?: UrlSelectFocusable
+  ) => void;
+  getNameFromId: (id: string) => string;
+  noHighlight?: boolean;
+}
+
+const UrlSelectItemGroup: React.FC<UrlSelectItemGroupProps> = ({
```

Comment:
we typically use `function()` syntax rather than arrow functions in the codebase.